### PR TITLE
[doc]docker and typo

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
This pr made the following changes:
- add trouble shooting EN
- update docker usage in get started
- fix lint and typos
- add cn docs

### CODE REVIEW

File: docs/manual-CN/1.overview/2.quick-start/1.get-started.md [hunk 1]
```diff
@@ -1,50 +1,16 @@
...
-
-`nebula` 镜像下载完成后，运行容器：
-
-```sh
-> docker run -it vesoft/nebula-graph:nightly /bin/bash
-```
+### 连接 Nebula Graph
 
-进入容器后，默认在根目录下 `/` ，切换到 `Nebula` 主目录:
+编译完成后，默认在根目录下 `/` ，切换到 `Nebula` 主目录:
```
Comment (yixinglu) [inline docs/manual-CN/1.overview/2.quick-start/1.get-started.md]: 编译完成后切换到 `Nebula` 默认安装目录：

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/create-tag-edge-syntax.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# CREATE TAG / EDGE 语法
+# CREATE TAG / EDGE 语句
```
Comment (yixinglu) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/create-tag-edge-syntax.md]: 这里还是 语法 更合适

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-edge-syntax.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
-```
+# DROP EDGE 语句
```
Comment (yixinglu) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-edge-syntax.md]: 还是语法更合适

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-space-syntax.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
-```
+# DROP SPACE 语句
```
Comment (yixinglu) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-space-syntax.md]: ditto

File: docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md [hunk 5]
```diff
@@ -1,4 +1,6 @@
-```
+# DROP TAG 语句
```
Comment (yixinglu) [inline docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/drop-tag-syntax.md]: ditto

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-syntax.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# Delete Syntax
+# DELETE 语句
```
Comment (yixinglu) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-syntax.md]: ditto

File: docs/manual-CN/3.build-develop-and-administration/1.build/build-by-docker.md [hunk 7]
```diff
@@ -1 +1,2 @@
-To be provided.
\ No newline at end of file
+# 通过 Docker 构建
+请参照 [Docker 构建](https://github.com/vesoft-inc/nebula-docker-compose/blob/master/README.md).
```
Comment (yixinglu) [inline docs/manual-CN/3.build-develop-and-administration/1.build/build-by-docker.md]: 这个要链接到 dev-docker 的 repo: https://github.com/vesoft-inc/nebula-dev-docker/blob/master/README.md

File: docs/manual-CN/3.build-develop-and-administration/1.build/build-source-code.md [hunk 8]
```diff
@@ -12,38 +13,41 @@ Nebula 基于 C++14 开发，因此它需要一个支持 C++14 的编译器。
 当编译类型为 **Debug** 的时候，最好预留 **30G** 磁盘空间
 
 ### 本地构建
+
 #### 步骤 1: 克隆代码
 
-```
+```sh
```
Comment (yixinglu) [inline docs/manual-CN/3.build-develop-and-administration/1.build/build-source-code.md]: shell

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administration/server-administration/storage-service-administration/data-import/download-and-ingest-sst-file.md [hunk 9]
```diff
@@ -10,7 +10,7 @@ Nebula 提供了 `Spark-SSTFile-Generator` 工具。
 
 执行后会在 `HDFS` 上生成 SST 文件，目录结构如下：
 
-```
+```plain
```
Comment (yixinglu) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administration/server-administration/storage-service-administration/data-import/download-and-ingest-sst-file.md]: text

File: docs/manual-EN/1.overview/2.quick-start/1.get-started.md [hunk 10]
```diff
@@ -2,25 +2,15 @@
...
-
-When `nebula` image is ready, run a container:
-
-```sh
-> docker run -it vesoft/nebula-graph:latest /bin/bash
-```
+### Connect to Service
 
-After login, you're in the `root` directory and you should switch to the nebula directory
+After compiling, you're in the `root` directory and you should switch to the nebula directory
```
Comment (yixinglu) [inline docs/manual-EN/1.overview/2.quick-start/1.get-started.md]: switch to the nebula default installed directory

File: docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 11]
```diff
@@ -1 +1,52 @@
...
+- Add the following value in nebula-storaged.conf and nebula-graphd.conf before starting the service
+
+   ```sh
+   --load_data_interval_secs=1
+   ```
+
+- Run the following command when starting the console
+
+   ```SQL
+   nebula> UPDATE VARIABLES graph:load_data_interval_secs=1
```
Comment (yixinglu) [inline docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md]: UPDATE CONFIG ?

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administration/server-administration/storage-service-administration/data-import/download-and-ingest-sst-file.md (hunk 9 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administration/server-administration/storage-service-administration/data-import/download-and-ingest-sst-file.md]: i think the two are same

File: docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md (hunk 11 above)
Comment (Amber1990Zhang) [reply docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md]: the code pr is not merged yet.

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administration/server-administration/storage-service-administration/storage-metrics.md [hunk 12]
```diff
@@ -31,15 +34,18 @@ _error_qps
...
 ### 时间范围
+
 时间范围目前只支持三种，分别为60，600，3600，分别表示最近一分钟，最近十分钟和最近一小时。
 
 ## 通过 HTTP 接口获取相应的性能指标
+
 根据上面的介绍，就可以写出一个完整的指标名称了，下面是一些示例：
 
-```
+```plain
```
Comment (yixinglu) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administration/server-administration/storage-service-administration/storage-metrics.md]: text

Comment (yixinglu) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administration/server-administration/storage-service-administration/storage-metrics.md]: text

File: docs/manual-CN/1.overview/2.quick-start/1.get-started.md [hunk 13]
```diff
@@ -1,50 +1,16 @@
 # Quick Start
 
-## Docker 方式
+## 通过 Docker
 
-Docker 安装方法请参考 [Docker 官方文档](https://docs.docker.com/)。
+通过 Docker 使用 Nebula，请参照[Nebula Docker 试用](https://github.com/vesoft-inc/nebula-docker-compose/blob/master/README.md)。
 
-从 Docker 镜像仓库获取最新 [Nebula Graph镜像](https://hub.docker.com/r/vesoft/nebula-graph)：
+## 通过源码
```
Comment (whitewum) [inline docs/manual-CN/1.overview/2.quick-start/1.get-started.md]: quick start 里面不需要通过源代码方式。只需要docker就行。这里改一下就行。

对于beginner，用docker 方式就够了。

**Comment by dangleptr** [comment]
Jenkins go

**Comment by whitewum** [comment]
4e26faf7e985e7c287daaf5c110ecd01108383eb merge in errors

**Pull request merged**